src/debugger/include/debugger/cli_debugger.h

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file cli_debugger.h
+ * Declares the CliDebugger Class for CLI Debugger Operations
+ */
+
+#pragma once
+
+#include "debugger.h"
+#include "debugger/debugger.fwd.h"
+#include "debugger/debugger_interface.h"
+
+namespace debugger {
+
+class CliDebugger : public IDebugger {
+  private:
+	std::shared_ptr<DebuggerCore> debugger_core;
+
+	std::string str;
+	std::string command_type;
+	std::string command;
+	std::string command_value;
```

Comment:
These can be local variables inside the `run_command` method

Reply:
Yeah. I don't need those variables any more.